### PR TITLE
prepare to release v0.4.5

### PR DESCRIPTION
**Pull request by cbgbt**
**Description of changes:**
* `cargo update`
* move a few dependencies to workspace that were missed in the initial move
* update twoliter version

**Testing done:**
* built bottlerocket core-kit & images in bottlerocket-os/bottlerocket
* unit & integration tests

**Terms of contribution:**

By submitting this pull request, I agree that this contribution is dual-licensed under the terms of both the Apache License, version 2.0, and the MIT license.

### CODE REVIEW

**Review by yeazelm — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -9,6 +9,21 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 [unreleased]: https://github.com/bottlerocket-os/twoliter/compare/v0.4.4...HEAD
 
+## [0.4.5] - 2024-08-21
```
Comment (KCSesh) [inline CHANGELOG.md]: Nit: Date

```suggestion
## [0.4.5] - 2024-08-22
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -9,6 +9,21 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
...
+
+### Changes
+
+- Update twoliter to re-resolve workspaces at buildtime to detect lock mismatches ([#337])
+- Improve logging in twoliter lockfile resolution ([#338])
+- pubsys: improve error messages on SSM parameter validation failure ([#348])
```
Comment (KCSesh) [inline CHANGELOG.md]: Nit Style:
```suggestion
- Improve error messages on SSM parameter validation failure for pubsys ([#348])
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -9,6 +9,21 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
...
+- pubsys: improve reliability of SSM parameter validation with client-side rate-limiting and retries ([#348])
```
Comment (KCSesh) [inline CHANGELOG.md]: Nit Style: 

```suggestion
- Improve reliability of SSM parameter validation with client-side rate-limiting and retries for pubsys ([#348])
```

**Review by KCSesh — APPROVED** [review]

File: CHANGELOG.md (hunk 3 above)
Comment (cbgbt) [reply CHANGELOG.md]: I wrote it this way to be consistent with the rest of this changelog, but am ok changing this.

**Comment by cbgbt** [comment]
^ fix suggestions from @KCSesh

**Review by yeazelm — APPROVED** [review]

**Review by KCSesh — APPROVED** [review]

**Review by bcressey — APPROVED** [review]

**Comment by cbgbt** [comment]
I've only made changes to the README with the last push. I'm going to override the Rust build workflow, which passed on the previous revision.

**Pull request merged**